tutorial/tutorial-gabc-01.html

```diff
@@ -89,7 +89,7 @@
 </pre>
 </div>
 
-<p>Here's the result after processing by gregorio and LaTeX:</p>
+<p>Here's the result after processing by gregorio and <span class="latex">L<span class="alpha">a</span>T<span class="epsilon">e</span>X</span>:</p>
```

Comment:
Since this is the first code example, everything in it is new so if anything gets highlighted in red, the whole thing should be highlighted.  Personally, I don't think that's entirely necessary.